pinot-common/src/main/java/org/apache/pinot/common/datablock/BaseDataBlock.java

```diff
@@ -27,6 +27,7 @@
 import java.util.HashMap;
 import java.util.Map;
 import javax.annotation.Nullable;
+import org.apache.pinot.common.datatable.DataTable;
```

Comment:
nit: we can copy the impl over. in fact we were deliberately decoupling data table from data block.

Reply:
Actually I'm going to pull `DataTable$CustomObject` to the top level, it's used in the serde so I can't just copy it.